lib/ansible/parsing/mod_args.py

```diff
@@ -178,15 +178,15 @@ def _normalize_parameters(self, thing, action=None, additional_args=dict()):
 
         return (action, final_args)
 
-    def _normalize_old_style_args(self, thing, action):
+    def _normalize_new_style_args(self, thing, action):
         '''
-        deals with fuzziness in old-style (action/local_action) module invocations
+        deals with fuzziness in new style module invocations
+        accepting key=value pairs and dictionaries, and always returning dictionaries
         returns tuple of (module_name, dictionary_args)
```

Comment:
The return comment doesn't match the return value. Not a new problem, but should be fixed.